ccd-definition/ComplexTypes.json

```diff
@@ -1 +1,155 @@
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "type",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "ApplicantType",
+    "ElementLabel": "Claimant type",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "individualTitle",
+    "FieldType": "Text",
+    "ElementLabel": "Title",
+    "FieldShowCondition": "type = \"INDIVIDUAL\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "individualFirstName",
+    "FieldType": "Text",
+    "ElementLabel": "First name",
+    "FieldShowCondition": "type = \"INDIVIDUAL\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "individualLastName",
+    "FieldType": "Text",
+    "ElementLabel": "Last name",
+    "FieldShowCondition": "type = \"INDIVIDUAL\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "individualDateOfBirth",
+    "FieldType": "Date",
+    "ElementLabel": "Date of birth",
+    "FieldShowCondition": "type = \"INDIVIDUAL\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "companyName",
+    "FieldType": "Text",
+    "ElementLabel": "Name",
+    "FieldShowCondition": "type = \"COMPANY\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "organisationName",
+    "FieldType": "Text",
+    "ElementLabel": "Name",
+    "FieldShowCondition": "type = \"ORGANISATION\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "soleTraderTitle",
+    "FieldType": "Text",
+    "ElementLabel": "Title",
+    "FieldShowCondition": "type = \"SOLE_TRADER\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "soleTraderFirstName",
+    "FieldType": "Text",
+    "ElementLabel": "First name",
+    "FieldShowCondition": "type = \"SOLE_TRADER\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "soleTraderLastName",
+    "FieldType": "Text",
+    "ElementLabel": "Last name",
+    "FieldShowCondition": "type = \"SOLE_TRADER\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "soleTraderTradingAs",
+    "FieldType": "Text",
+    "ElementLabel": "Trading as",
+    "FieldShowCondition": "type = \"SOLE_TRADER\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "soleTraderDateOfBirth",
+    "FieldType": "Date",
+    "ElementLabel": "Date of birth",
+    "FieldShowCondition": "type = \"SOLE_TRADER\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "Applicant",
+    "ListElementCode": "applicantAddress",
+    "FieldType": "AddressUK",
+    "ElementLabel": "Address",
+    "SecurityClassification": "Public"
```

Comment:
When it comes to addresses, email, and phone numbers, the specific purpose should be part of the name. i.e. applicantHomeAddress, applicantCorrespondenceAddress,

Reply:
if you define what these are I am happy to add, I believe the process is to add to a separate ticket, within another PR I have changed to primaryAddress